include/logger/loggerConfig.h

```diff
@@ -357,17 +358,19 @@ typedef struct _GPSConfig {
 #define DEFAULT_GPS_SATELLITE_CONFIG {"GPSSats", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
 #define DEFAULT_GPS_QUALITY_CONFIG {"GPSQual", "", 0, 5, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
 #define DEFAULT_GPS_DOP_CONFIG {"GPSDOP", "", 0, 20, DEFAULT_GPS_SAMPLE_RATE, 1, 0}
-
-#define DEFAULT_GPS_CONFIG {                   \
-		DEFAULT_GPS_LATITUDE_CONFIG,           \
-		DEFAULT_GPS_LONGITUDE_CONFIG,          \
-		DEFAULT_GPS_SPEED_CONFIG,              \
-		DEFAULT_GPS_DISTANCE_CONFIG,           \
-		DEFAULT_GPS_ALTITUDE_CONFIG,           \
-		DEFAULT_GPS_SATELLITE_CONFIG,          \
-		DEFAULT_GPS_QUALITY_CONFIG,            \
-		DEFAULT_GPS_DOP_CONFIG                 \
-         }
+#define DEFAULT_GPS_HEADING_CONFIG {"GPSHeading", "Degrees", 0, 360, DEFAULT_GPS_SAMPLE_RATE, 0, 0}
```

Comment:
I chose to avoid the word "Track" for reasons that I hope are obvious.  That is why I figured we would compare two headings... one for GPS and one for magnetic.  I could call it track if you like... it may be confusing to customers but its dead on accurate.  I figured doing two headings would be easier to understand.